### PR TITLE
Use fixed timestamps in Minify transform

### PR DESCRIPTION
**Pull request by liutikas**

### CODE REVIEW

**Review by jbartok — CHANGES_REQUESTED** [review]

**Comment by jbartok** [comment]
Pls. base this PR on the `release` branch so that we can include in Gradle 7.6.

File: build-logic/basics/src/main/kotlin/gradlebuild/basics/classanalysis/AnalyzeAndShade.kt [hunk 1]
```diff
@@ -162,7 +162,9 @@ class JarAnalyzer(
 
 
 fun JarOutputStream.addJarEntry(entryName: String, sourceFile: File) {
-    putNextEntry(ZipEntry(entryName))
+    val entry = ZipEntry(entryName)
+    entry.time = 0
```
[the remark this answers is not in the document]
Comment (liutikas) [reply build-logic/basics/src/main/kotlin/gradlebuild/basics/classanalysis/AnalyzeAndShade.kt]: Done

**Comment by liutikas** [comment]
> Pls. base this PR on the `release` branch so that we can include in Gradle 7.6.

Done

**Comment by liutikas** [comment]
Anything else missing here?

**Review by jbartok — APPROVED** [review]

**Comment by jbartok** [comment]
@bot-gradle test this

**Comment by jbartok** [comment]
@bot-gradle test and merge

**Comment by jbartok** [comment]
@bot-gradle test and merge

**Comment by jbartok** [comment]
@bot-gradle test and merge

**Comment by jbartok** [comment]
@bot-gradle test and merge

**Pull request merged**